renv/scripts/common/activate

```diff
@@ -8,16 +8,6 @@ deactivate () {
         export PATH
         unset _OLD_VIRTUAL_PATH
 
-        rm "$HOME/.Rprofile"
```

Comment:
@santina I do hesitate to remove this, but we can figure out how to do it in python at some other point.